src/api/records.tsx

```diff
@@ -109,18 +109,24 @@ export const useRecords = <T extends unknown = Record[]>(
         resultsPerPage: number;
         sort: SortType;
         dateRange: DateRange;
+        plotRecords: boolean;
       }
     ]
-  >
+  >,
+  plotRecords = false
```

Comment:
Could name this more generically as `paginate` - as that more accurately represents what this variable is controlling